tests/test_experiment_rsmtool_1.py

```diff
@@ -43,6 +46,53 @@ def test_run_experiment_parameterized(*args, **kwargs):
     check_run_experiment(*args, **kwargs)
 
 
+def test_run_experiment_lr_with_notebook_rerun():
+    # basic experiment with LinearRegression model and notebook
+    # run-run after the experiment after `RSM_REPORT_DIR` is deleted
+    # to ensure that the `.environ.json` file can be located
+
+    source = 'lr-with-notebook-rerun'
+    experiment_id = 'lr'
+    config_file = join(rsmtool_test_dir,
+                       'data',
+                       'experiments',
+                       source,
+                       '{}.json'.format(experiment_id))
+    do_run_experiment(source, experiment_id, config_file)
+
+    report_ipynb = join('test_outputs', source, 'report', '{}_report.ipynb'.format(experiment_id))
+    report_html = join('test_outputs', source, 'report', '{}_report.ipynb'.format(experiment_id))
+
+    del os.environ['RSM_REPORT_DIR']
+
+    Reporter.convert_ipynb_to_html(report_ipynb, report_html)
```

Comment:
add `check_report(html_report)` to make sure there are no errors/warnings within the report?